recipes/opentdf-client/all/conanfile.py

```diff
@@ -15,7 +15,7 @@ class OpenTDFConan(ConanFile):
     generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
     options = {"build_python": [True, False], "fPIC": [True, False]}
-    default_options = {"build_python": False, "fPIC": True}
+    default_options = {"build_python": False, "fPIC": True, "libzip:with_openssl": False, "libarchive:with_zlib":False}
```

Comment:
Value of the options is encoded in the package-id, so if you change it, Conan will compute a different package-id for those requirements (`libzip` and `libarchive`) and it might be missing.

In this CI we are building a bunch of configurations, but all of them use the default option values. So, if you change some of these defaults, the computed package ID won't be available.

Are these option values required?